src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -228,5 +228,5 @@
 @mixin hxButton-hover--tertiary {
   background-color: transparent;
   border-color: transparent;
-  color: $cyan-500;
+  color: $cyan-700;
```

Comment:
Please update to the new `v2.0` color variables:

https://github.com/HelixDesignSystem/helix-ui/blob/eb547a4294af4f768294ffef085cfa58136718c1/src/scss/_vars.scss#L1-L118

Reply:
Changes have been done, Please review